akka-actor/src/main/scala/akka/pattern/StatusReply.scala

```diff
@@ -47,6 +47,21 @@ final class StatusReply[+T] private (private val status: Try[T]) {
   def isError: Boolean = status.isFailure
   def isSuccess: Boolean = status.isSuccess
 
+  /**
+   * Return this StatusReply as a Try.
+   */
+  def toTry: Try[T] = status
+
+  /**
+   * Return this StatusReply as an Either.
+   */
+  def toEither: Either[Throwable, T] = status.toEither
+
+  /**
+   * Return this StatusReply as an Option, returning None and discarding the error if it's an Error.
+   */
+  def toOption: Option[T] = status.toOption
```

Comment:
Add `Scala API: ` to the docs of all 3 methods:
```
/**
   * Scala API: Return this StatusReply as an Option, returning None and discarding the error if it's an Error.
   */
```

Also add a `def toOptional` for Java.